compile fail when using async_drop

<!--
Thank you for filing a bug report! 🐛 Please provide a short summary of the bug,
along with any information you feel relevant to replicating the bug.
-->

in my [project](https://github.com/Sherlock-Holo/lycoris/tree/4ec655b4a00e36c46ceaa109f52786f1623fec28)

I enable the async_drop feature in client workspace, then I got compile error

I expected to see this happen: *compile success*

Instead, this happened: *compile error*

### Meta
<!--
If you're using the stable version of the compiler, you should also check if the
bug also exists in the beta or nightly versions.
-->

`rustc --version --verbose`:
```
rustc 1.81.0-nightly (d7f6ebace 2024-06-16)
binary: rustc
commit-hash: d7f6ebacee13b6c03623c4b74197280454ede8de
commit-date: 2024-06-16
host: x86_64-unknown-linux-gnu
release: 1.81.0-nightly
LLVM version: 18.1.7
```

<!--
Include a backtrace in the code block by setting `RUST_BACKTRACE=1` in your
environment. E.g. `RUST_BACKTRACE=1 cargo build`.
-->
<details><summary>Backtrace</summary>
<p>

```
   Compiling lycoris-client v0.3.1 (/home/sherlock/git/lycoris/client)
error: linking with `cc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/self-contained:/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/self-contained:/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/self-contained:/home/sherlock/.cargo/bin:/home/sherlock/.nix-profile/bin:/home/sherlock/.local/bin:/home/sherlock/go/bin:/home/sherlock/.cargo/bin:/home/sherlock/.nix-profile/bin:/home/sherlock/.local/bin:/home/sherlock/go/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/home/sherlock/.local/share/flatpak/exports/bin:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/usr/lib/rustup/bin" VSLANG="1033" "cc" "-m64" "/tmp/rustcvVS6Tg/symbols.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0127dc2d0mucv8xc7355nh0wk.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.02uev4ch4mud319h3i0yh5sn9.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.034fqyepdh86ikpif8msmuief.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0al9il3836xhexagf572d6ypu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0fnh4x9ch3eybjhpe8239mezz.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0gnr3vlesu9d3hsd9omkpk003.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0h25h6vtvp46dzi9ttgaa3et0.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0k04mwrigspe2go7ax8g7ath3.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0n6oj8rw9bdxsa5rsgkuc64zg.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0ogpo65nktshwsc4o0bvumwtu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0t8vltcnrgt1tkwpqbhflg3yb.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0ufnkekhj0gq0l99gr9mnrjs7.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.0wn2xdtlk5cm7g5lzvucvf2i6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.10pnjkt2vllras5bavi9cy61w.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.12e05yk9mdqbdskd11c3s4ueo.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.139yrpn6v1ojacqi2sdxf051g.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.14n5cb57psh44vy0fk5qhgr29.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.157bv22dv34v4od7ejhg9c43w.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.15ln5jrr0c52updurtzzq0me5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.15nrj5ni8em195bs356zdhp1s.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.17q6jsajlnd1c7uysxjz5xohw.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.18azun2uyvnx2qcqz4e88ycpr.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1bkdprf7kag6v419l84xi8a8m.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1cwsf2xng3fe9xkr972padsf2.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1dyrr42smyujd2ee88c9yu29u.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1jxxfqdyhosr59mmsylxvh6x5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1l250fu2g7cc5g5ai1pvf7x2z.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1l5tdufbzdxzvt43j7l8mxxeu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1lkcxa1spgrkmlwxd6vcbvjw9.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1ozoc0tm3uh3mfbgvu4mvdlli.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1q06wfnlx9rjvid8g097u9ycm.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1r6as9hoo84kxwma5sokr9bap.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1r9ajvl42lzuqsuqmf0b9kqqy.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1rj1sw1kvp6r4ypoba6o0pjbg.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1t2oqnsu5msysoaf4nudk8u9g.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1tt534i3h0lq3jaam2raz7tb5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1uo8xehkqwt52i9nmy7nz768h.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1vm2h1kwldi53ezx5zx2muesc.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1wls4haetq335bwr4b24e5grr.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.1ymh30hco3xock2pimof4jhiu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.20gaeu8le29hyxys6tixgaai6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.21fph3ny9izdy589fbgwsv4fa.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.21z7zq9h48r6557clq968a8o7.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.25vwv9cdem30n4mno7tyn6qlt.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.27bcix9gevobcfbcp071vgxqq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.27naz4i267ifu86oj8wdljxdd.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2a7gwdqc8l3x53gunk5xoayxu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2ad9vozq2i5xixwh4l1dbj7jj.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2blndt962lr0uex4ugke8idk4.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2c5y4u6lzzsbgp38mfg6adms8.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2ezt32z7c6ve2qytdihmp3cd8.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2g7t9qa4mw6inirfg3em2ruu2.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2gkhywn2uwbwpql4gh8du880k.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2h0odbezet15pr932ll4y0vqr.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2jguamko2k2sbiwzpsv6o8zrc.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2jsll8imv8v4ppqorx4qejk6o.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2nhsnw3loakf4ddrxefk839k1.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2o6n4glkjob3ufl1blzk151v3.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2p1kmw2gt9dt70nb7h15drmwq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2v3r7gs1r33j16ce93qffn524.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2yqz2u1nie8j0xldl6itl0yvf.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.2z6j6zv278et4abha9u0xfrwm.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.30oj6hi4mo65ued1kkvtg9b2e.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.326xcyt6rbj8sc7o7nj9bixn6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.32extpu12w59d4odvn4nc395l.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.35u68gicc293zym8riyr70d08.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.36dbanmdedfvcvkzhyorhbxpy.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.38hts9jl9anvby0f6hsiig7s4.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.39n4hunqqzd16wzhg2o5siuif.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3b9l241cqch54dr97tug67zz6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3enik4x3qwxcdb9wyhnwr1zcq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3fwoiod9xnede5mj3jpnbfrlk.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3gdg9zx6v2j9dgi8m1x97qq1d.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3ldlhc6g4dxc4czb1nz76ng9x.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3tayuptbjauj5rt20is7uw1vu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3ur3w98i2kl3zko5growoaz1c.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.3x9vvinql47i8rkx27jf69oix.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.41irlcj0nalupkoidoahiscw5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.42jdm3t8hejwkqa7ef4bfdjbq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.47sefr0q9h7ib9dbu5dpfy6mi.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.49mqw5bxi2uxke2nu5hsf1e16.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.4eg22wn8pl48rh9negn94d4nz.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.4f8ua9l5bm9sr4tqhqebnepc6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.4kkww62jb1q09vq0663ztlcji.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.4rshe6es48m8mcz3z1b9f6j9x.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.4vril6ywmcj6rfla1t9qc57m9.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.4x7zv0czhd29m3fl02qcs5hyq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.52ep57y7psvpwmalb4htuuadq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.52ioiunsjx74riwcixzf9x6e0.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.55616xh8ed9rtrylvnbmoumoq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.55rld22yygj5urxodg5l122nw.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.55uhrrde9r84ycluacdii9xtw.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5a01w6sqkj1op17z0pesss6w7.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5aq3b9dhiicoa373gb32m2hck.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5cdtlbyplvpg4penfh4ecv2bc.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5h55kd22ugqxk62q6fywj3tcy.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5ipm2ye0s2m3ha6v75jhsja6p.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5mhxpn6ks5bb2i4ukckrbycz5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5n17bwtfv7qu8sy4nq1te5cc8.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5pgv0zq6g9294ebr8ddmbazvv.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5qdgukgjsv9sobcnsb5yc3nlh.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5qn6yhutbhihaqbcg60w6kh9h.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5tms3h5asy7p8t0rdneovxbld.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.5zuattbskcnq4ui6hs2kzs5c6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.60cu9h6pt13cn3chix8qde28b.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.61bjoew1vf7epefza7s51dg1h.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.61ltoh4oxfihtlu3ztkulexiu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6705cborneiqv3iykk3ej4i5d.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.673ajpift3il19mzrezzfys77.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6fyeooeamept4rz9rl6qszov1.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6g9w6dm71bu43saxma74x4vnb.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6htk2aautmybc1q8vxm1qjczl.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6jq1aw1ck6pa4xo3wu9wqfhii.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6l1josh0w9q2ql0diij8ui83v.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6n1rr0j5wadjbx2m050s0z83j.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6nipzspbtse9qdvwb4go5ainf.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6o733na7wh7nw51yhlkzdnt84.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6q705qmv5y4w8hcp3hmfgly6g.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6r9mm9j88eliqytp5lic3pon5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6rzgrxzsjbivsh4adejrd2dk9.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6tqztx7340koh6otm4vngk0q3.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6uu56thywrkc1m5dbyoh00o0q.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6x83ybc5delt6vnwso408eguf.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.6xlf81w6yrcdf7dsvmoddktyk.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.70yk57trq2c6c470kbgy708mi.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.728220t1o61pxzj3mb5nohcpv.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.79oiswmm3s4c9e7bzh6iqwl3n.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7a77iw5kqrakjekva1u4adzgm.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7ajezo6e6m9ie3zt1ackrouat.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7ak9je3pz8tsp58u45px66r6y.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7c6jskn01qfnnsz95at3o2xz0.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7ffrskj75j62xgcd36embou0n.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7gviz0l3pudy9laa5eiu8qr86.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7hv7icheipkpvuyo4rif9675n.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7ndkpv4pvau8nz84f419ipqin.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7ohy9rjzxmf3031fdhq3s49eh.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7ujuznnqmo8szeboxv535skba.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7vbodom36jdfq07p1plnd8foa.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.7w6e0jnopg1sswzz2wiawace4.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8035r3usqlmsjgn1ek2xfeuq8.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.83egyi4qufm3oa4v48wruhole.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.83k1kq5lsc31dgpzg9ln6cdbw.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.874pdj8sgs1ate1zd63tky14r.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8asrr5flxfcety9azpmd6ed92.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8dl5o54v0gnga8q3ebsr4trmv.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8iz90m45zvuflkzdxymntapye.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8jamsesiio4u1k2ungor5r05l.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8niw0x07hwuet7qq7wxv5tqvu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8ry5x3fcppcsd1pbn85grlsja.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8t8y0uxz0t1po7mqdhdwbpzqn.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8u1ee6p3mzjj1q0kqj0fxf3o8.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8uk0r7xel3llvyzqog6bwriyr.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8ve3tuwkl6hu6hqd2c6q8z41w.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.92esr41uiayk26mr3zrz835wi.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.92q81n2dhoa2k7i8a6e6b2o9b.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.955fheovq27fj4rnc7otxuazx.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.99uh69j2rqpsqpp4fo7cb6z5y.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9cm7gzlcfk78eedt5roo6w0p3.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9f52gqtook0ate00q3jhg82cu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9fsejetf07ilhqf5lev1ixn76.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9gtqejqkx39m3ibcd04qcp17j.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9koarcltad6fqflg4zh3jsgru.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9mrpi8mmw4idern0vqrgkgd8y.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9pdewagel83o8dzfvxgeplb2w.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9sbr7f64lo1872a26mm1aajn1.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9vgwn88cgk4smkjon6521knzc.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9xrffw7k74cy0xhull1vd2qzn.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9y7fwih02idvobxthovqfjvw5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.9y838x3i3z20vr5k5m56tt2bj.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.a009kkq52j2jmormltqrams7d.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.a08zf5xjknkamwmlnaedhw19w.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.a5a2iai0bhvf2cszueqvo6tfb.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.a6y6kq6dmjny3r8333ir6kvi3.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.a6zwa8gktq81ua06kxypsu09e.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.a8wkruek9340oxxiq436msfwt.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.aex553dbdtefwyoqdi59vq0ct.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ag84n6xoxtn34girvqk50c6np.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.alwo1evj362vlywj2ccvr2aup.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ao9lc56cyefnf3q999vmic8ds.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ar2xqcn2su5ucqx3b16hzlaxa.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ar3ses6u1u8slaot5wy39p3l6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.atfa4kjlag8aakx5nqxxe5x7s.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.aur41j19olj6diiuyfdwhu44k.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.av6oogkh9i8wktg2skojfodqq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.av8a1nrdr7ehkqct8j8jv8hnd.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.awr0d0zlmxgbl0wbbucwize40.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.axk7i11w78k2mbke05uhv6z24.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.azs9bf0twaqfm6o23srqbp2k7.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bgu9anlm3ydkm3ulo3ln8towk.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bguc3djitndix1uv503y6zaun.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bi9obk2tjygnk6wjwycf337y4.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.blelm7gmnpy6ycvtlcld96qk2.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bnpze0wkxwr2f4m7exjv8efx5.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bp5mot6t56k64sm47psi5n8i7.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bq99zy699cjcd6ojj9l9qgk69.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bqbr2uxsv8l5iyx1tofnfjhiy.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bqo72vpyd5fvy6ricfdctmcrh.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.br27qyi3th68j9v93aq1nha6z.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.bv0nyxismybewvh2pdrmkfkmi.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.c143vj0zbfhwglvwskliyivcn.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.c1t291doelqz21lqk80jyhcwc.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.c431fw530e6rsxxztzp7esk8d.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.c4ror2k2004fiwzjwarq1bi56.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.c5nhrzkpaeyd0gduoehxehpc6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.c8ppes0uklvqnw24ig66jfvq0.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.c9pixlj3m1cqqvm2olekcc2as.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.cahqopq3v2ohyxan5ziyik77q.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.cfp09ood5u3nzb5p21e54qvdu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.cfs6cbwy2mdl5z21o8aarh19f.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.chlza6hal0d2qo3mkefxo15yn.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.clnytirk5xwfi88mb446f1ywv.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.cm8vlazft30it5xgfdcp2s41o.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.cn2rbtkyzmxfp1oxvxodiyyn9.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.cncj3akkuf6yx23n6u0v3x4fr.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.cnm78rr3cz8sjc8eki5algdew.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.d1bq34nigxou2w4mim3q3qgzc.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.d4agvnlp04jjhmn4vqy7n9lfi.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.d4wo0943g0cieu29q95nsrwpe.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.d7c399xj69k2mgfmeb4n7lpy6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.d7jd4ahw8fgz8pj7oq237i37g.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.d9obb12zsxofb42q305rljlpz.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.deb6mcwyccpx5imfclhvwv5ne.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dmaa7zpj7j4ehrq9aww66njqy.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dnhxn4iuyd666o2w3hldo4qy8.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dnz0x0gwhcgfq7jdvv6ne7ebz.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dowgeqaosvc903qalxbuw6pu4.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dozea11b10ocssyj2ykeyo0jj.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dp6udjz7v7xdb8m3oljqwjsvo.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dqqru5qjro1pzfh1un7x7troo.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dtxski28psddozaqsdnleacy9.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.du5i9rgq1ad5jx6mqa47ykwyf.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dvojtbtymijcdlfvjcxchs2lr.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.dxpopcv6ge0mwpcyyqj4twhnj.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.e5jsdgm832re0e7hw6u15lvro.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ee89tecebadq10yfxhjlkjwlk.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.eetzz3n0t8vy40ks4sedwpmn3.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.efhbny7tsqap2ggqj5vzyo2v6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.efqzmoexpjej2ze110wixud55.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.eg1i7zh2eakh37myo0b9faz96.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.eg6kbutytnaakrxwy9lofz1s1.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.el22twskxzl8wr8jk9mb2h5k7.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.elsa3tgfacc7icrmqf2kic2az.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.eo0kdbt1jbx73e23n3vcabx8f.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.epb96u1g24xbnss7l3r7v1tg8.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.eprwprbr3ydnnxalw1n4kdfaw.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ertotsqbuntfi7uy2vrg0k7rq.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.esskn5ecrxr9h0qf72ldotyy2.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.eun088bcprooy2qpsiqyj45i2.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ev1v04scc15hfxi98b95rzob6.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ew1dhghtv2depx4aicij2u5fp.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.eyxgey4q6qvy5m7rka0dbzztu.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.ezqvm0cpnrxmlxwssqwgx10fn.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.f2u1pfmhe8t1qv7ol4e1unlg4.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.f48coqw9q67l6b7s607d42pzz.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.f48qvz8vi2nhmzogyh8obtkvk.rcgu.o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.44p6as08nf1pv7u5rd1os4z74.rcgu.o" "-Wl,--as-needed" "-L" "/home/sherlock/git/lycoris/target/debug/deps" "-L" "/home/sherlock/git/lycoris/target/debug/build/aws-lc-sys-f64e314d8a40adda/out" "-L" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/home/sherlock/git/lycoris/target/debug/deps/liblycoris_client-af1b932b16737725.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librustls_native_certs-3e99ea7ba3b279ab.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librustls_pemfile-18b4e90a0d9b6774.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libbase64-1e08e3f71b4fd6d9.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libopenssl_probe-7a88c39453af999d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libserde_yaml-a8bbe78f7c3d5f67.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libryu-2ae7af5af0bb88e0.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libunsafe_libyaml-18c724e223d22ffd.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librtnetlink-61cd8a50a3cccf25.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnix-e3bdae2038f7f59f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnetlink_proto-3dc57fd5e7112014.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnetlink_sys-b8e24506882fa335.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures-8cbc3aab33eb92cf.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_executor-ebf74a3c1c01a941.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnetlink_packet_route-bca78e635997efb6.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnetlink_packet_core-c04e4aaef23b016d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnetlink_packet_utils-110f9d8e4dcc84d4.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libbyteorder-921e9eb05e03d6ac.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libserde-158eddcbe4a40151.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtokio_stream-cbffbcbb07749890.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libshare-b3bf70706a297ed1.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtap-4aa41518670171b0.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtracing_subscriber-7f0ca5cbebd6ef35.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnu_ansi_term-539d6c8f66d1b861.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liboverload-667acc2908675926.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libthread_local-bea2067a721757e6.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libsharded_slab-288a256a379985cf.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtracing_log-1a80a529db2aa770.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libprotocol-248983a87eb8dd1d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtotp_rs-67bd7b57d27b2813.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liburlencoding-e70222ec02c08ce8.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libsha2-a1727ad833f5e4a2.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libsha1-f943e43641bc6b49.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libcpufeatures-e58d67928e727c66.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhmac-9751dd4d4bb266aa.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libdigest-0262ad1feb3415d5.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libblock_buffer-a7767f52d5be9300.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libcrypto_common-94609bf7600bb847.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libgeneric_array-07414a2bcb428e6d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtypenum-5326f72807cfe834.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libconstant_time_eq-16f1bc2993f8ebd6.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libbase32-8d097b8908ee8163.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhttp_body_util-0bfbacc25f0c7f9c.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhyper_util-33e09af08057c9ff.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtower-f5933075f08900cd.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libpin_project-3d28680475c4b2e5.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtower_layer-59d9149cea32c325.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtower_service-56141c8379744c53.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhyper-92f5832b90ac137f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libwant-e685d7a406130720.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtry_lock-c81c639303c73d15.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libh2-e09fd20b40ef2566.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libindexmap-a1dfeb0d7c9ae11f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libequivalent-9e89cbd86631a94f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhashbrown-95174c35c7134cbd.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libahash-d746a58a3cef3faa.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libzerocopy-c5da16b22d4d8428.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liballocator_api2-8e4287854e2d1221.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libatomic_waker-139ffb07fa982694.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtokio_util-77568a4dc8e528bd.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhttpdate-cec10d4fe91818dd.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhttp_body-e07ab925367c5791.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhttp-59f5385987ae9b1c.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libitoa-add5260810035b6e.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfnv-574dedeaa4c89e1d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhickory_resolver-4650a8964d2691c0.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libresolv_conf-20b7666dcded2a4a.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhostname-6602120870fc882e.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libmatch_cfg-4d64094482ccd140.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libquick_error-9804de68ea34bdae.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libparking_lot-ece4650a6562e4c8.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libparking_lot_core-1e539c5492ede417.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liblock_api-4bfa6e5802bfd45b.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libscopeguard-003ee3b9b09cebdb.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liblru_cache-110cbe7fb7d6713f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liblinked_hash_map-4155900399b65b23.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libhickory_proto-f1e4c20930fbaaac.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librand-4ba4d0be7f3c11b0.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librand_chacha-eba0c0e050243b5c.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libppv_lite86-905fd835b337451d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librand_core-77889abe69efb228.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libgetrandom-838c6bdbc612a754.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libcfg_if-57bf25c23e31efdb.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libdata_encoding-732fa069be813c83.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liburl-e6742392478890b8.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libidna-1b68c433942a066d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_normalizer-fcb1fecb5ec33a1a.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_normalizer_data-062659b7008575dd.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libwrite16-d18c02dcb762aeae.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libutf8_iter-7b46fdc8748b2bd7.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libutf16_iter-e8333c715a5bfde4.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libsmallvec-2bf71e8326cc0d46.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_properties-b8bf1a34790e6a98.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_properties_data-03b5a57048e77534.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_locid_transform-6ba03793138bc41d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_locid_transform_data-d55fb9d2ae757896.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_collections-ab21318872d18293.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_provider-17842de54b480732.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libicu_locid-03fe96f6ddae4557.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liblitemap-05df7b5877556460.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtinystr-10ec1ffad2843f30.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libzerovec-796a3e55c674bad4.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libwriteable-4ad35b37dd154aab.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libyoke-dd05ad43c7f2363c.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libzerofrom-4ad1d9bfbf4fd947.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libstable_deref_trait-94bc1fab9293791f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libform_urlencoded-07a2dedb236e9598.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libpercent_encoding-a400589552762a21.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libipnet-e48bd8e2986d8365.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libidna-467bc400a08cef4b.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libunicode_normalization-2de07759203e0547.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libunicode_bidi-41651475181819c6.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtinyvec-a28cafa3c8699823.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtinyvec_macros-66661c55905090ec.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtracing-3e46fb7b5376b02b.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtracing_core-2051afaedd8a2e9d.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_util-910fa49fac790cd6.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libslab-f7057b79dbdd9024.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_channel-2be897bb93b482b7.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_sink-2288d7d2e7cd0bb1.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_task-31d564b3124d8637.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libpin_utils-76c2785263cdb1c1.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_core-bbe118dd79718c74.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_rustls-a7f6b27e207c01cb.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librustls-4f342273d104c169.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libsubtle-29dc240e5dede6d0.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libwebpki-d3df05dbccf41ec5.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libuntrusted-0d4f2f5eacbcc494.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libaws_lc_rs-50ce82df3be30af5.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libmirai_annotations-048fbe9565197efa.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libaws_lc_sys-bc0171ff6de4355f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libonce_cell-795d6939e3fe8636.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libzeroize-d697f47119386afc.rlib" "/home/sherlock/git/lycoris/target/debug/deps/librustls_pki_types-85fb22a2e41c3ce5.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libfutures_io-daea6097586892ae.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libclap-b084a012405b3cd5.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libclap_builder-6b389cdee03b0bb2.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libstrsim-0c998b0290db27b9.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libanstream-2cdedfd307f759e7.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libanstyle_query-5a024fef37b4b230.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libis_terminal_polyfill-b597e53a3abf9b93.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libcolorchoice-59ec4f279830c8c8.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libanstyle_parse-a238c0b917528828.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libutf8parse-d435e47da5544871.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libclap_lex-4e06e4c45b975ae5.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libanstyle-a9f033361773f9f4.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libcidr-af33cb2297fe2956.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libaya_log-e645200759866f11.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libaya_log_common-2fe7ef43c9f8a030.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnum_enum-8b2bf4f265da0f6a.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libaya-48b958d7c74754fb.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libbitflags-587d8e547c38cd0e.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liblazy_static-b82c15677deef9e2.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libassert_matches-87316ac06bd88b01.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libaya_obj-03f6618dadd86c02.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libthiserror-b231a67a4059bb5f.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libobject-a309e057d0c494a0.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libmemchr-18a8ca001d702c17.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liblog-a00a231d50da9d95.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libanyhow-fddfbf3ae1c518e4.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libtokio-7fdc8ea2865c4dd7.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libsignal_hook_registry-e38a53dc0fbae64a.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libnum_cpus-cd0ea41b1a724c54.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libsocket2-a44307394dda862b.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libbytes-30dacc79d7337c17.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libmio-a248dda548665ae7.rlib" "/home/sherlock/git/lycoris/target/debug/deps/liblibc-98dbae3fd83efd2b.rlib" "/home/sherlock/git/lycoris/target/debug/deps/libpin_project_lite-0917a66d1286620d.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-9f4544d1e1cf245e.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-5c2158c1682442e0.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libobject-404d0b40d67de09f.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libmemchr-e07eeabf48db885c.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libaddr2line-c92423f648579064.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgimli-8dab5299a97564ae.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-65891f932666bb65.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd_detect-b2634d236d77b88a.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libhashbrown-38ef4997455a5e72.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_alloc-b2e6b39d803e65fe.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libminiz_oxide-670f2ef8c6803edf.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libadler-e599f337a82d59c4.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-acbe32895031f6bc.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcfg_if-e21e611894cf2844.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-7b90a2705bcf265b.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-1f7e19b44251c73e.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-f72b956e24d1de70.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-632ae0f28c5e55ff.rlib" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-4d7d16bbf0636a40.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-B/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/gcc-ld" "-B/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/gcc-ld" "-B/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin/gcc-ld" "-fuse-ld=lld" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-L" "/home/sherlock/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/self-contained" "-o" "/home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-nodefaultlibs"
  = note: rust-lld: error: undefined symbol: core::future::async_drop::async_drop_in_place_raw$LT$tokio..runtime..task..join..JoinHandle$LT$$LP$$RP$$GT$$GT$::h2de059ca182ad3c9
          >>> referenced by async_drop.rs:92 (/rustc/d7f6ebacee13b6c03623c4b74197280454ede8de/library/core/src/future/async_drop.rs:92)
          >>>               /home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o:(core::future::async_drop::async_drop_in_place::h054908401aa827b9)

          rust-lld: error: undefined symbol: core::future::async_drop::async_drop_in_place_raw$LT$futures_channel..mpsc..UnboundedSender$LT$netlink_proto..protocol..request..Request$LT$netlink_packet_route..message..RouteNetlinkMessage$C$futures_channel..mpsc..UnboundedSender$LT$netlink_packet_core..message..NetlinkMessage$LT$netlink_packet_route..message..RouteNetlinkMessage$GT$$GT$$GT$$GT$$GT$::hbeecc53e9f06f1c0
          >>> referenced by async_drop.rs:92 (/rustc/d7f6ebacee13b6c03623c4b74197280454ede8de/library/core/src/future/async_drop.rs:92)
          >>>               /home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o:(core::future::async_drop::async_drop_in_place::h2315df8d97d9c59b)

          rust-lld: error: undefined symbol: core::future::async_drop::async_drop_in_place_raw$LT$rtnetlink..handle..Handle$GT$::h7b2ae191c949b4c6
          >>> referenced by async_drop.rs:92 (/rustc/d7f6ebacee13b6c03623c4b74197280454ede8de/library/core/src/future/async_drop.rs:92)
          >>>               /home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o:(core::future::async_drop::async_drop_in_place::h27ce1eb414cb3448)

          rust-lld: error: undefined symbol: core::future::async_drop::async_drop_in_place_raw$LT$netlink_proto..handle..ConnectionHandle$LT$netlink_packet_route..message..RouteNetlinkMessage$GT$$GT$::hd121095cdf2700b9
          >>> referenced by async_drop.rs:92 (/rustc/d7f6ebacee13b6c03623c4b74197280454ede8de/library/core/src/future/async_drop.rs:92)
          >>>               /home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o:(core::future::async_drop::async_drop_in_place::h88bed92295b5c597)

          rust-lld: error: undefined symbol: core::future::async_drop::async_drop_in_place_raw$LT$futures_channel..mpsc..UnboundedSenderInner$LT$netlink_proto..protocol..request..Request$LT$netlink_packet_route..message..RouteNetlinkMessage$C$futures_channel..mpsc..UnboundedSender$LT$netlink_packet_core..message..NetlinkMessage$LT$netlink_packet_route..message..RouteNetlinkMessage$GT$$GT$$GT$$GT$$GT$::h4882f434c57da74b
          >>> referenced by async_drop.rs:92 (/rustc/d7f6ebacee13b6c03623c4b74197280454ede8de/library/core/src/future/async_drop.rs:92)
          >>>               /home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o:(core::future::async_drop::async_drop_in_place::h8a4d1a536316f865)

          rust-lld: error: undefined symbol: core::future::async_drop::async_drop_in_place_raw$LT$core..option..Option$LT$futures_channel..mpsc..UnboundedSenderInner$LT$netlink_proto..protocol..request..Request$LT$netlink_packet_route..message..RouteNetlinkMessage$C$futures_channel..mpsc..UnboundedSender$LT$netlink_packet_core..message..NetlinkMessage$LT$netlink_packet_route..message..RouteNetlinkMessage$GT$$GT$$GT$$GT$$GT$$GT$::hb216e3e152c515cc
          >>> referenced by async_drop.rs:92 (/rustc/d7f6ebacee13b6c03623c4b74197280454ede8de/library/core/src/future/async_drop.rs:92)
          >>>               /home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o:(core::future::async_drop::async_drop_in_place::hb8c7fa873ed1e3cb)

          rust-lld: error: undefined symbol: core::future::async_drop::async_drop_in_place_raw$LT$alloc..sync..Arc$LT$futures_channel..mpsc..UnboundedInner$LT$netlink_proto..protocol..request..Request$LT$netlink_packet_route..message..RouteNetlinkMessage$C$futures_channel..mpsc..UnboundedSender$LT$netlink_packet_core..message..NetlinkMessage$LT$netlink_packet_route..message..RouteNetlinkMessage$GT$$GT$$GT$$GT$$GT$$GT$::h96ad548310e457f7
          >>> referenced by async_drop.rs:92 (/rustc/d7f6ebacee13b6c03623c4b74197280454ede8de/library/core/src/future/async_drop.rs:92)
          >>>               /home/sherlock/git/lycoris/target/debug/deps/lycoris_client-2061ef1eaad0cd0b.8lp58mfmb6ce44569mmoaxbbk.rcgu.o:(core::future::async_drop::async_drop_in_place::heb33ddecc3c17bc0)
          collect2: error: ld returned 1 exit status


error: could not compile `lycoris-client` (bin "lycoris-client") due to 1 previous error
```

</p>
</details>

I also try to make a minimum reproducibility, but I failed, https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=5c98906a97da2afc87de3a2edd72a0fe this codes can build with same rustc version

--- a/compiler/rustc_codegen_ssa/src/back/symbol_export.rs
+++ b/compiler/rustc_codegen_ssa/src/back/symbol_export.rs
@@ -371,7 +371,7 @@ fn exported_symbols_provider_local(
                 }) => {
                     // A little sanity-check
                     debug_assert_eq!(
-                        args.non_erasable_generics(tcx, def_id).skip(1).next(),
+                        args.non_erasable_generics(tcx, def_id).next(),
                         Some(GenericArgKind::Type(ty))
                     );
                     symbols.push((
@@ -422,10 +422,7 @@ fn upstream_monomorphizations_provider(
                 }
                 ExportedSymbol::AsyncDropGlueCtorShim(ty) => {
                     if let Some(async_drop_in_place_fn_def_id) = async_drop_in_place_fn_def_id {
-                        (
-                            async_drop_in_place_fn_def_id,
-                            tcx.mk_args(&[tcx.lifetimes.re_erased.into(), ty.into()]),
-                        )
+                        (async_drop_in_place_fn_def_id, tcx.mk_args(&[ty.into()]))
                     } else {
                         // `drop_in_place` in place does not exist, don't try
                         // to use it.
@@ -480,6 +477,17 @@ fn upstream_drop_glue_for_provider<'tcx>(
     }
 }
 
+fn upstream_async_drop_glue_for_provider<'tcx>(
+    tcx: TyCtxt<'tcx>,
+    args: GenericArgsRef<'tcx>,
+) -> Option<CrateNum> {
+    if let Some(def_id) = tcx.lang_items().async_drop_in_place_fn() {
+        tcx.upstream_monomorphizations_for(def_id).and_then(|monos| monos.get(&args).cloned())
+    } else {
+        None
+    }
+}
+
 fn is_unreachable_local_definition_provider(tcx: TyCtxt<'_>, def_id: LocalDefId) -> bool {
     !tcx.reachable_set(()).contains(&def_id)
 }
@@ -491,6 +499,7 @@ pub fn provide(providers: &mut Providers) {
     providers.upstream_monomorphizations = upstream_monomorphizations_provider;
     providers.is_unreachable_local_definition = is_unreachable_local_definition_provider;
     providers.upstream_drop_glue_for = upstream_drop_glue_for_provider;
+    providers.upstream_async_drop_glue_for = upstream_async_drop_glue_for_provider;
     providers.wasm_import_module_map = wasm_import_module_map;
     providers.extern_queries.is_reachable_non_generic = is_reachable_non_generic_provider_extern;
     providers.extern_queries.upstream_monomorphizations_for =

--- a/compiler/rustc_codegen_ssa/src/back/symbol_export.rs
+++ b/compiler/rustc_codegen_ssa/src/back/symbol_export.rs
@@ -470,22 +470,16 @@ fn upstream_drop_glue_for_provider<'tcx>(
     tcx: TyCtxt<'tcx>,
     args: GenericArgsRef<'tcx>,
 ) -> Option<CrateNum> {
-    if let Some(def_id) = tcx.lang_items().drop_in_place_fn() {
-        tcx.upstream_monomorphizations_for(def_id).and_then(|monos| monos.get(&args).cloned())
-    } else {
-        None
-    }
+    let def_id = tcx.lang_items().drop_in_place_fn()?;
+    tcx.upstream_monomorphizations_for(def_id)?.get(&args).cloned()
 }
 
 fn upstream_async_drop_glue_for_provider<'tcx>(
     tcx: TyCtxt<'tcx>,
     args: GenericArgsRef<'tcx>,
 ) -> Option<CrateNum> {
-    if let Some(def_id) = tcx.lang_items().async_drop_in_place_fn() {
-        tcx.upstream_monomorphizations_for(def_id).and_then(|monos| monos.get(&args).cloned())
-    } else {
-        None
-    }
+    let def_id = tcx.lang_items().async_drop_in_place_fn()?;
+    tcx.upstream_monomorphizations_for(def_id)?.get(&args).cloned()
 }
 
 fn is_unreachable_local_definition_provider(tcx: TyCtxt<'_>, def_id: LocalDefId) -> bool {

--- a/compiler/rustc_middle/src/query/mod.rs
+++ b/compiler/rustc_middle/src/query/mod.rs
@@ -1554,12 +1554,13 @@ rustc_queries! {
         }
     }
 
-    /// The entire set of monomorphizations the local crate can safely link
-    /// to because they are exported from upstream crates. Do not depend on
-    /// this directly, as its value changes anytime a monomorphization gets
-    /// added or removed in any upstream crate. Instead use the narrower
-    /// `upstream_monomorphizations_for`, `upstream_drop_glue_for`, or, even
-    /// better, `Instance::upstream_monomorphization()`.
+    /// The entire set of monomorphizations the local crate can safely
+    /// link to because they are exported from upstream crates. Do
+    /// not depend on this directly, as its value changes anytime
+    /// a monomorphization gets added or removed in any upstream
+    /// crate. Instead use the narrower `upstream_monomorphizations_for`,
+    /// `upstream_drop_glue_for`, `upstream_async_drop_glue_for`, or,
+    /// even better, `Instance::upstream_monomorphization()`.
     query upstream_monomorphizations(_: ()) -> &'tcx DefIdMap<UnordMap<GenericArgsRef<'tcx>, CrateNum>> {
         arena_cache
         desc { "collecting available upstream monomorphizations" }
@@ -1599,6 +1600,26 @@ rustc_queries! {
     ///       (like `Clone::clone` for example).
     query upstream_drop_glue_for(args: GenericArgsRef<'tcx>) -> Option<CrateNum> {
         desc { "available upstream drop-glue for `{:?}`", args }
+    }
+
+    /// Returns the upstream crate that exports async-drop-glue for
+    /// the given type (`args` is expected to be a single-item list
+    /// containing the type one wants async-drop-glue for).
+    ///
+    /// This is a subset of `upstream_monomorphizations_for` in order
+    /// to increase dep-tracking granularity. Otherwise adding or
+    /// removing any type with async-drop-glue in any upstream crate
+    /// would invalidate all functions calling async-drop-glue of an
+    /// upstream type.
+    ///
+    /// You likely want to call `Instance::upstream_monomorphization()`
+    /// instead of invoking this query directly.
+    ///
+    /// NOTE: This query could easily be extended to also support other
+    ///       common functions that have are large set of monomorphizations
+    ///       (like `Clone::clone` for example).
+    query upstream_async_drop_glue_for(args: GenericArgsRef<'tcx>) -> Option<CrateNum> {
+        desc { "available upstream async-drop-glue for `{:?}`", args }
     }
 
     /// Returns a list of all `extern` blocks of a crate.

--- a/compiler/rustc_middle/src/ty/instance.rs
+++ b/compiler/rustc_middle/src/ty/instance.rs
@@ -219,8 +219,9 @@ impl<'tcx> Instance<'tcx> {
             InstanceKind::Item(def) => tcx
                 .upstream_monomorphizations_for(def)
                 .and_then(|monos| monos.get(&self.args).cloned()),
-            InstanceKind::DropGlue(_, Some(_)) | InstanceKind::AsyncDropGlueCtorShim(_, _) => {
-                tcx.upstream_drop_glue_for(self.args)
+            InstanceKind::DropGlue(_, Some(_)) => tcx.upstream_drop_glue_for(self.args),
+            InstanceKind::AsyncDropGlueCtorShim(_, Some(_)) => {
+                tcx.upstream_async_drop_glue_for(self.args)
             }
             _ => None,
         }
@@ -256,7 +257,7 @@ impl<'tcx> InstanceKind<'tcx> {
         match self {
             ty::InstanceKind::Item(def) => Some(def),
             ty::InstanceKind::DropGlue(def_id, Some(_))
-            | InstanceKind::AsyncDropGlueCtorShim(def_id, _)
+            | InstanceKind::AsyncDropGlueCtorShim(def_id, Some(_))
             | InstanceKind::ThreadLocalShim(def_id) => Some(def_id),
             InstanceKind::VTableShim(..)
             | InstanceKind::ReifyShim(..)
@@ -267,6 +268,7 @@ impl<'tcx> InstanceKind<'tcx> {
             | ty::InstanceKind::ConstructCoroutineInClosureShim { .. }
             | ty::InstanceKind::CoroutineKindShim { .. }
             | InstanceKind::DropGlue(..)
+            | InstanceKind::AsyncDropGlueCtorShim(..)
             | InstanceKind::CloneShim(..)
             | InstanceKind::FnPtrAddrShim(..) => None,
         }
@@ -330,6 +332,26 @@ impl<'tcx> InstanceKind<'tcx> {
                 adt_def
                     .destructor(tcx)
                     .map_or_else(|| adt_def.is_enum(), |dtor| tcx.cross_crate_inlinable(dtor.did))
+            });
+        }
+        if let ty::InstanceKind::AsyncDropGlueCtorShim(.., Some(ty)) = *self {
+            // Async drop glue generally wants to be instantiated at
+            // every codegen unit, but without an #[inline] hint. We
+            // should make this available to normal end-users.
+            if tcx.sess.opts.incremental.is_none() {
+                return true;
+            }
+            // When compiling with incremental, we can generate a *lot* of
+            // codegen units. Including drop glue into all of them has a
+            // considerable compile time cost.
+            //
+            // We include enums without destructors to allow, say, optimizing
+            // drops of `Option::None` before LTO. We also respect the intent of
+            // `#[inline]` on `Drop::drop` implementations.
+            return ty.ty_adt_def().map_or(true, |adt_def| {
+                adt_def
+                    .async_destructor(tcx)
+                    .map_or_else(|| adt_def.is_enum(), |dtor| tcx.cross_crate_inlinable(dtor.ctor))
             });
         }
         if let ty::InstanceKind::ThreadLocalShim(..) = *self {

--- a/compiler/rustc_middle/src/ty/instance.rs
+++ b/compiler/rustc_middle/src/ty/instance.rs
@@ -314,7 +314,9 @@ impl<'tcx> InstanceKind<'tcx> {
         if self.requires_inline(tcx) {
             return true;
         }
-        if let ty::InstanceKind::DropGlue(.., Some(ty)) = *self {
+        if let ty::InstanceKind::DropGlue(.., Some(ty))
+        | ty::InstanceKind::AsyncDropGlueCtorShim(.., Some(ty)) = *self
+        {
             // Drop glue generally wants to be instantiated at every codegen
             // unit, but without an #[inline] hint. We should make this
             // available to normal end-users.
@@ -329,29 +331,14 @@ impl<'tcx> InstanceKind<'tcx> {
             // drops of `Option::None` before LTO. We also respect the intent of
             // `#[inline]` on `Drop::drop` implementations.
             return ty.ty_adt_def().map_or(true, |adt_def| {
-                adt_def
-                    .destructor(tcx)
-                    .map_or_else(|| adt_def.is_enum(), |dtor| tcx.cross_crate_inlinable(dtor.did))
-            });
-        }
-        if let ty::InstanceKind::AsyncDropGlueCtorShim(.., Some(ty)) = *self {
-            // Async drop glue generally wants to be instantiated at
-            // every codegen unit, but without an #[inline] hint. We
-            // should make this available to normal end-users.
-            if tcx.sess.opts.incremental.is_none() {
-                return true;
-            }
-            // When compiling with incremental, we can generate a *lot* of
-            // codegen units. Including drop glue into all of them has a
-            // considerable compile time cost.
-            //
-            // We include enums without destructors to allow, say, optimizing
-            // drops of `Option::None` before LTO. We also respect the intent of
-            // `#[inline]` on `Drop::drop` implementations.
-            return ty.ty_adt_def().map_or(true, |adt_def| {
-                adt_def
-                    .async_destructor(tcx)
-                    .map_or_else(|| adt_def.is_enum(), |dtor| tcx.cross_crate_inlinable(dtor.ctor))
+                match *self {
+                    ty::InstanceKind::DropGlue(..) => adt_def.destructor(tcx).map(|dtor| dtor.did),
+                    ty::InstanceKind::AsyncDropGlueCtorShim(..) => {
+                        adt_def.async_destructor(tcx).map(|dtor| dtor.ctor)
+                    }
+                    _ => unreachable!(),
+                }
+                .map_or_else(|| adt_def.is_enum(), |did| tcx.cross_crate_inlinable(did))
             });
         }
         if let ty::InstanceKind::ThreadLocalShim(..) = *self {

--- a/compiler/rustc_sanitizers/src/cfi/typeid/itanium_cxx_abi/transform.rs
+++ b/compiler/rustc_sanitizers/src/cfi/typeid/itanium_cxx_abi/transform.rs
@@ -288,6 +288,7 @@ pub fn transform_instance<'tcx>(
     mut instance: Instance<'tcx>,
     options: TransformTyOptions,
 ) -> Instance<'tcx> {
+    // FIXME: account for async-drop-glue
     if (matches!(instance.def, ty::InstanceKind::Virtual(..))
         && tcx.is_lang_item(instance.def_id(), LangItem::DropInPlace))
         || matches!(instance.def, ty::InstanceKind::DropGlue(..))

